R/attach.R

```diff
@@ -1,4 +1,4 @@
-core <- c("crane", "gtsummary")
+core <- c("crane", "gtsummary", "cards", "cardx")
```

Comment:
Using {crane} a bit for testing, I think it is best to show them too

Reply:
Added!

![image](https://github.com/user-attachments/assets/840a97fa-54af-490e-a878-0e17d3c05746)

Reply:
i really like this feature. if time allows, i think we should make it available to all NEST packages.